### ISSUE TITLE
Allow importing & exporting of user color schemes.

### ISSUE THREAD

**Issue by AObuchow** [AObuchow/Eclipse-Modern-Dark-Theme#54]
As mentioned in https://github.com/AObuchow/Eclipse-Modern-Dark-Theme/issues/52#issue-639028119 & https://github.com/AObuchow/Eclipse-Modern-Dark-Theme/issues/46#issuecomment-643937552, it'd be nice to be able to save & load user-defined color schemes. 

The user could have a list of saved color schemes to switch between in the preference page (depends on #52).

I'm not sure yet what file type & format should be used when a color scheme is exported. I was thinking perhaps JSON due to its simplicity or CSS. An advantage of using CSS is that if hex codes are used to encode the color scheme's colors, their color preview will be shown in the Generic Editor.

**Comment by AObuchow** [comment]
@ingomohr, @PyvesB, @mbooth101: do any of you have an opinion or suggestion on this?

**Comment by AObuchow** [comment]
Here's what I had in mind (if CSS is used):
```CSS
.ModernDark {
	background-color: #232629;
	base-color: #2A2E32;
	accent-color: #AA0000;'
}
```

In the Generic Editor, this gets rendered as:
![image](https://user-images.githubusercontent.com/10300119/84726627-515aa080-af5b-11ea-8856-83e0f8c0f528.png)

**Comment by ingomohr** [comment]
I like the CSS-option, too. That way, you could not just export the theme but also review and edit it right away.

**Comment by mbooth101** [comment]
> do any of you have an opinion or suggestion on this?

Theme is already stored as CSS, why introduce an extra intermediary format? As @ingomohr suggests, this way you can simply have a "Open in CSS Editor" button instead of crafting a complex preferences dialog panel :-)

**Comment by AObuchow** [comment]
Agreed, CSS seems to be the best way to store these color schemes :) thanks for the feedback everyone

**Comment by AObuchow** [comment]
When this is implemented, a note should be made about using this theme plugin in combination with Wild Web Developer to enable in-editor previews of the CSS colours.

**Comment by AObuchow** [comment]
Slight improvement on my original CSS suggestion:

CSS has support for [custom properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) which are indicated by prefixing a property with `--`.

So, a more valid CSS to represent Modern Dark color schemes would be:
```CSS
ModernDark {
	--background-color: #232629;;
	--base-color: #2A2E32;
	--accent-color: #AA0000;
}
```

Which will be rendered as the following in the Generic Editor (note the lack of warning markers):
![image](https://user-images.githubusercontent.com/10300119/84977158-a6cbb480-b0f7-11ea-836a-87510541b797.png)
